assets/annuaire/info.vue

```diff
@@ -252,6 +258,9 @@ export default {
         },
     },
     methods: {
+        retour(){
+            this.$router.go(-1)
+        },
```

Comment:
En général, c'est une excellente idée de séparer la logique de l'interface. Cependant, dans ce cas la logique de `this.$router.go(-1)` est déjà minimale. En faite, toute la complexité est déjà faite derrière la fonction `go()`. Du coup, je dirais qu'il n'est pas nécessaire de rajouter une méthode «juste» pour ça et donc appeler `$router.go(-1)` directement dans le composant `<b-btn>`.